changes-set.txt

```diff
@@ -85,6 +85,15 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+15-Feb-25 abeq2f    eqabf
+15-Feb-25 rabeq2i   reqabi
+15-Feb-25 abeq1i    eqabci
+15-Feb-25 abeq2i    eqabi
+15-Feb-25 abeq2d    eqabd
+15-Feb-25 abeq2w    eqabw
+15-Feb-25 abeq2impr eqab1r      Reverse direction of abeq
+15-Feb-25 abeq1     eqabc       Commuted form of abeqb
```

Comment:
```suggestion
15-Feb-25 abeq1     eqabc       Commuted form of eqab
```
Use "eqab" in the notes since at that point, abeq2 has been relabeled to eqab (it is on the following line, and lines are in decreasing chronological order).

Also, I removed the line concerning relabeling of `abeq2impr` since that result is a handful of days old, so no need to clutter changes-set.txt with it.